src/components/App/App.jsx

```diff
@@ -1,16 +1,25 @@
 import './App.css';
 import React from 'react';
 import PropTypes from 'prop-types';
+// import Navigation from '../Navigation/Navigation.jsx'
+import { Routes, Route, Link } from 'react-router-dom';
 
-function App() {
+export default function App() {
   return (
     <div className="App">
-      Content TBD
+      <header>
+        {/* <Link to={'/'}><h1>Tavern Keeper</h1></Link> */}
+        {/* <Navigation /> */}
+      </header>
+      <Routes>
+        {/* <Route path="/" element={<Home />}/>
```

Comment:
I'm seeing warnings in the console because router is partially set up but without actually having the route for '/' matched. I think we should fix that before merging this in. 

Here's how we can do that:
1. Uncomment this line 15 in the App.jsx file
2. Add `import Home from '../Home/Home';` to this App.jsx file
3. Add 'Home' in the Home.jsx component by adding some baseline jsx:
    ```
    const Home = () => {
      return (
        <div className='Home'></div>
      );
    };
    ```
4. And then export it at the bottom of the Home.jsx file by adding `export default Home;`

Reply:
Can do, I opted not to do that in the first place cuz I was worried about getting off track for the scope of the PR but that makes sense to resolve this error.